### PR TITLE
SYS-1502: Implement homepage redesign

### PR DESCRIPTION
**Pull request by ztucker4**
Implements [SYS-1502](https://uclalibrary.atlassian.net/browse/SYS-1502)
Figma link: https://www.figma.com/file/R6n38mi5qmVnkS4FWHhrLS/Feature-Redesigns?node-id=2726%3A7468&mode=dev

This PR adds the redesigned Primo homepage. All changes should apply only to the homepage. However, I needed to restyle some possibly-reused spacing-related Primo elements (in lines 1-27 of `homepage.css`). Please take a look at other parts of Primo and double check for any apparent spacing changes.

I have added fewer "cards" in this implementation than shown in the Figma based on preliminary DFG feedback. And as in the reusable footer component, the footer contains fewer links here than in Figma. All links and visible text are placeholders for now. Future work is needed to accommodate smaller viewports.

[SYS-1502]: https://uclalibrary.atlassian.net/browse/SYS-1502?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by akohler — CHANGES_REQUESTED** [review]
@ztucker4 I'm unable to log into Primo when using the current branch, because the login box doesn't show any options:
SYS-1502_login_box

I couldn't quickly tell what from `homepage.css` might be overriding that box's CSS.

**Comment by ztucker4** [comment]
This is ready for re-review. I fixed the issue with login buttons by using a more specific CSS selector in line 3 of `homepage.css`.

**Comment by ztucker4** [comment]
Hi @akohler, this is ready for another look. The selectors in lines 15-24 are even more specific now and no longer hide the spacing divs in the Library Card pages.

File: views/01UCS_LAL-PRIMO_REDESIGN/css/ucla/homepage.css [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+}
+/* Hide side "spacer" divs in desktop view */
+@media (min-width: 1280px) {
+  prm-search md-content .flex-lg-25 {
+    display: none;
+  }
+}
+@media (min-width: 1280px) {
+  prm-search md-content .flex-lg-10 {
+    display: none;
```
Comment (akohler) [inline views/01UCS_LAL-PRIMO_REDESIGN/css/ucla/homepage.css:22]: @ztucker4 The margin problems have been fixed, except... on the "Journal Title" page, the left margin is present, but empty of the "Journals by category" list.  That list is present in the [deployed `PRIMO_REDESIGN` view](https://search.library.ucla.edu/discovery/jsearch?vid=01UCS_LAL:PRIMO_REDESIGN) and I see it when I check out the `main` branch locally, but not with this branch.  The culprit seems to be this `display: none;`.

Here's what I see with this branch:
primo_redesign_missing_journal_categories

**Review by akohler — CHANGES_REQUESTED** [review]
@ztucker4 The margin problems have been fixed, except... on the "Journal Title" page, the left margin is present, but empty of the "Journals by category" list.  That list is present in the [deployed `PRIMO_REDESIGN` view`(https://search.library.ucla.edu/discovery/jsearch?vid=01UCS_LAL:PRIMO_REDESIGN) and I see it when I check out the `main` branch locally, but not with this branch.

Here's what I see with this branch:

**Comment by ztucker4** [comment]
@akohler This is ready for another (hopefully final) look, with an even more specific selector.

**Comment by akohler** [comment]
@ztucker4 Progress, but not yet... The Journal Title facets do appear now at first... but after doing a search, they disappear:
primo_journal_title_margin

**Comment by ztucker4** [comment]
Hi @akohler, I've fixed the most recent issue with Browse facets. I also found and fixed a problem with `max-width` being applied incorrectly to the Citation Linker page (http://localhost:8003/discovery/citationlinker?vid=01UCS_LAL:PRIMO_REDESIGN).

**Review by akohler — APPROVED** [review]

**Comment by akohler** [comment]
@ztucker4 I can't find any more problems, and didn't even know about the citation linker page :)
All looks good to me, merging!

**Pull request merged**